collated/main/A0144061U.md

```diff
@@ -0,0 +1,606 @@
+# A0144061U
+###### /java/seedu/todoList/commons/events/model/DeadlineListChangedEvent.java
+``` java
+/** Indicates the TodoList in the model has changed*/
+public class DeadlineListChangedEvent extends BaseEvent {
+
+    public final ReadOnlyTaskList data;
+
+    public DeadlineListChangedEvent(ReadOnlyTaskList data){
+        this.data = data;
+    }
+
+    @Override
+    public String toString() {
+        return "number of tasks " + data.getTaskList().size();
+    }
+}
+```
+###### /java/seedu/todoList/commons/events/model/EventListChangedEvent.java
+``` java
+/** Indicates the TodoList in the model has changed*/
+public class EventListChangedEvent extends BaseEvent {
+
+    public final ReadOnlyTaskList data;
+
+    public EventListChangedEvent(ReadOnlyTaskList data){
+        this.data = data;
+    }
+
+    @Override
+    public String toString() {
+        return "number of tasks " + data.getTaskList().size();
+    }
+}
+```
+###### /java/seedu/todoList/commons/events/model/TodoListChangedEvent.java
+``` java
+/** Indicates the TodoList in the model has changed*/
+public class TodoListChangedEvent extends BaseEvent {
+
+    public final ReadOnlyTaskList data;
+
+    public TodoListChangedEvent(ReadOnlyTaskList data){
+        this.data = data;
+    }
+
+    @Override
+    public String toString() {
+        return "number of tasks " + data.getTaskList().size();
+    }
+}
+```
+###### /java/seedu/todoList/commons/events/storage/StorageLocationChangedEvent.java
+``` java
+/**
+ * Indicates to observers when the location of the storage is changed
+ */
+public class StorageLocationChangedEvent extends BaseEvent {
+
+    public String newDirectory;
+
+    public StorageLocationChangedEvent(String directory) {
+        this.newDirectory = directory;
+    }
+    
+    public String getNewDirectory() {
+    	return newDirectory;
+    }
+
+    @Override
+    public String toString(){
+        return "New Location: " + this.newDirectory;
+    }
+
+}
+```
+###### /java/seedu/todoList/logic/commands/ClearCommand.java
+``` java
+    @Override
+    public CommandResult execute() {
+        assert model != null;
+        switch(dataType) {
+        	case "todo":
```

Comment:
seems like you are using tabs for indentation; the coding guidelines require using 4 spaces for indentation